### PR TITLE
SG-30111 added to troubleshooting guide

### PR DESCRIPTION
**Pull request by shaynacohen**

### CODE REVIEW

File: docs/en/quick-answers/troubleshooting/module-tank-has-no-attribute-support-url.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+The following message appears when launching {% include product %} Desktop after upgrading the version:
+
+```
+{% include product %} Desktop Error:
+Error: module 'tank' has no attribute 'support_url'
+```
+
+## Causes
+
+'support_url' was introduced in tk-core v0.19.18. and the Descriptor version is incompatible with the newer {% include product %} Desktop version 1.7.3.
```
Comment (dsauve-adsk) [inline docs/en/quick-answers/troubleshooting/module-tank-has-no-attribute-support-url.md:21]: I think that the "'support_url' was introduced in tk-core v0.19.18." part is more an explanation than the cause itself and that's why I believe it should be displayed after

Like this: 
The Descriptor version is incompatible with the newer {% include product %} Desktop version 1.7.3.
'support_url' was introduced in tk-core v0.19.18.

**Review by dsauve-adsk — CHANGES_REQUESTED** [review]

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/1cadabc3da4d5061e4e0c5f717308cb20d474ab1/index.html)

File: docs/en/quick-answers/troubleshooting/module-tank-has-no-attribute-support-url.md (hunk 1 above)
Comment (shaynacohen) [reply docs/en/quick-answers/troubleshooting/module-tank-has-no-attribute-support-url.md]: Fixed, thanks @dsauve-adsk !

**Review by dsauve-adsk — APPROVED** [review]

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/413ec221fe4f000e302a1decd2ed45e7095584b1/index.html)

**Pull request merged**